timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/search/description/indexes/PersonIndexDescription.java

```diff
@@ -6,21 +6,23 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class PersonIndexDescription implements IndexDescription {
+class PersonIndexDescription implements IndexDescription {
```

Comment:
Kan ik nu makkelijk voor een ander soort persoon een index aanmaken die specifiek is voor die persoon?